labelbox/schema/model_run.py

```diff
@@ -50,10 +50,10 @@ def add_predictions(
         kwargs = dict(client=self.client, model_run_id=self.uid, name=name)
         if isinstance(predictions, str) or isinstance(predictions, Path):
             if os.path.exists(predictions):
-                return MEAPredictionImport.create_from_file(path=predictions,
-                                                            **kwargs)
+                return MEAPredictionImport.create_from_file(
+                    path=str(predictions), **kwargs)
             else:
-                return MEAPredictionImport.create_from_url(url=predictions,
+                return MEAPredictionImport.create_from_url(url=str(predictions),
```

Comment:
hmmm did this work prior? why do we need this change?

Reply:
It doesn't actually effect anything. It just silences mypy.